schema/projectSchema.js

```diff
@@ -7,33 +7,70 @@ const projectSchema = object({
     .max(100, "Name must be under 100 characters"),
   description: string().required("Description is required").max(500, "Description must be under 500 characters"),
   eulogy: string().required("Eulogy is required").max(1000, "Eulogy must be under 1000 characters"),
-  causeOfDeath: string().required("Cause of death is required").max(255, "Keep cause of death under 255 characters"),
-  startDate: date().required("Start date is required").typeError("Start date must be a valid date"),
-  endDate: date()
-    .required("End date is required")
-    .typeError("End date must be a valid date")
-    .when("startDate", (startDate, schema) =>
-      startDate ? schema.min(startDate, "End date must be after start date") : schema
-    ),
-  types: array().of(number()).min(1, "At least one type must be selected").required("Types are required"),
-  tombstoneId: number().nullable(),
-});
+  causeOfDeath: string().required("Cause of death is required").max(100, "Keep cause of death under 100 characters"),
```

Comment:
if keeping test as is, change to:

```suggestion
  causeOfDeath: string().required("Cause of death is required").max(255, "Keep cause of death under 255 characters"),
```

Reply:
updated the test instead I want smaller causes of death to fit on the cards better.